tfx/extensions/experimental/kfp_compatibility/kfp_container_component_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for tfx.extensions.experimental.kfp_compatibility.kfp_container_component."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+from tfx.dsl.component.experimental import placeholders, container_component
+from tfx.extensions.experimental.kfp_compatibility.kfp_container_component import load_kfp_yaml_container_component
+from tfx.types.experimental.simple_artifacts import File
+
+class KubeflowContainerComponentTest(tf.test.TestCase):
```

Comment:
You need to add the following to this file, so that the test is runnable.
``` 
if __name__ == '__main__':
  tf.test.main()
```

Reply:
Fixed! Thanks